clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -3,148 +3,177 @@
 // Test for Intel FPGA loop attributes applied not to a loop
 void foo() {
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::ivdep]] int a[10];
+  [[intel::ivdep]] int a[10];
+  // expected-error@+1 {{ loop attributes must be applied to for, while, or do statements}}
+  [[intel::ivdep(2)]] int b[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::ivdep(2)]] int b[10];
+  [[intel::ii(2)]] int c[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::ii(2)]] int c[10];
-  // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::max_concurrency(2)]] int d[10];
+  [[intel::max_concurrency(2)]] int d[10];
 
   int arr[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::ivdep(arr)]] int e[10];
+  [[intel::ivdep(arr)]] int e[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::ivdep(arr, 2)]] int f[10];
+  [[intel::ivdep(arr, 2)]] int f[10];
 
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::disable_loop_pipelining]] int g[10];
+  [[intel::disable_loop_pipelining]] int g[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::loop_coalesce(2)]] int h[10];
+  [[intel::loop_coalesce(2)]] int h[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::max_interleaving(4)]] int i[10];
+  [[intel::max_interleaving(4)]] int i[10];
   // expected-error@+1 {{intelfpga loop attributes must be applied to for, while, or do statements}}
-  [[intelfpga::speculated_iterations(6)]] int j[10];
+  [[intel::speculated_iterations(6)]] int j[10];
+}
+
+// Test for deprecated spelling of Intel FPGA loop attributes
+void foo_deprecated() {
+  int a[10];
+  // expected-warning@+2 {{attribute 'ivdep' is deprecated}}
```

Comment:
I think this diagnostic is a bit strange because the attribute mention doesn't match with actual spelling in the code. Can it say that spelling is deprecated or include the full spelling string as it was written in source code, i.e. like this:
```suggestion
  // expected-warning@+2 {{attribute 'intelfpga::ivdep' is deprecated}}
```

Reply:
Done